models/uniswap/ethereum/uniswap_ethereum_airdrop_claims.sql

```diff
@@ -21,7 +20,7 @@ WITH early_price AS (
     , MIN_BY(price, minute) AS price
     FROM {{ source('prices', 'usd') }}
     WHERE blockchain = 'ethereum'
-    AND contract_address= {{uni_token_address}}
+    AND contract_address = {{uni_token_address}}
```

Comment:
This will compiled into `AND contract_address='0x1f9840a85d5af5bf1d1762f925bdaddc4201f984'`

I will try `from_hex('{{uni_token_address}}')`